### PR TITLE
Add font size customization

### PR DESCRIPTION
**Pull request by LunarX**
Depends on #3

### CODE REVIEW

File: rich-html-editor/src/main/java/com/infomaniak/lib/richhtmleditor/JsBridge.kt [hunk 1]
```diff
@@ -38,17 +39,19 @@ internal class JsBridge(
 
     fun removeFormat() = execCommand(OtherCommand.REMOVE_FORMAT)
 
-    fun setTextColor(@ColorInt color: Int) = execCommand(StatusCommand.TEXT_COLOR, colorToRgbHex(color))
+    fun setTextColor(color: JsColor) = execCommand(StatusCommand.TEXT_COLOR, color)
 
-    fun setTextBackgroundColor(@ColorInt color: Int) = execCommand(StatusCommand.BACKGROUND_COLOR, colorToRgbHex(color))
+    fun setTextBackgroundColor(color: JsColor) = execCommand(StatusCommand.BACKGROUND_COLOR, color)
+
+    fun setFontSize(@IntRange(from = 1, to = 7) fontSize: Int) = execCommand(StatusCommand.FONT_SIZE, fontSize)
```
Comment (KevinBoulongne) [inline rich-html-editor/src/main/java/com/infomaniak/lib/richhtmleditor/JsBridge.kt:46]: Magic numbers.

File: rich-html-editor/src/main/java/com/infomaniak/lib/richhtmleditor/RichHtmlEditorWebView.kt [hunk 2]
```diff
@@ -112,8 +113,10 @@ class RichHtmlEditorWebView @JvmOverloads constructor(
     fun toggleStrikeThrough() = jsBridge.toggleStrikeThrough()
     fun toggleUnderline() = jsBridge.toggleUnderline()
     fun removeFormat() = jsBridge.removeFormat()
-    fun setTextColor(@ColorInt color: Int) = jsBridge.setTextColor(color)
-    fun setTextBackgroundColor(@ColorInt color: Int) = jsBridge.setTextBackgroundColor(color)
+    fun setTextColor(@ColorInt color: Int) = jsBridge.setTextColor(JsColor(color))
+
```
Comment (KevinBoulongne) [inline rich-html-editor/src/main/java/com/infomaniak/lib/richhtmleditor/RichHtmlEditorWebView.kt:117]:
```suggestion
```

File: sample/src/main/java/com/infomaniak/lib/richhtmleditor/sample/EditorSampleFragment.kt [hunk 3]
```diff
@@ -80,6 +80,10 @@ class EditorSampleFragment : Fragment() {
         textColorBlue.setOnClickListener { editor.setTextColor(BLUE) }
         textBackgroundColorRed.setOnClickListener { editor.setTextBackgroundColor(RED) }
         textBackgroundColorBlue.setOnClickListener { editor.setTextBackgroundColor(BLUE) }
+
+        fontSmallButton.setOnClickListener { editor.setFontSize(2) }
+        fontMediumButton.setOnClickListener { editor.setFontSize(4) }
+        fontBigButton.setOnClickListener { editor.setFontSize(6) }
```
Comment (KevinBoulongne) [inline sample/src/main/java/com/infomaniak/lib/richhtmleditor/sample/EditorSampleFragment.kt:86]: Magic numbers.

**Review by KevinBoulongne — CHANGES_REQUESTED** [review]

File: rich-html-editor/src/main/java/com/infomaniak/lib/richhtmleditor/RichHtmlEditorWebView.kt (hunk 2 above)
Comment (LunarX) [reply rich-html-editor/src/main/java/com/infomaniak/lib/richhtmleditor/RichHtmlEditorWebView.kt:117]: How did this end up here? Thanks

File: rich-html-editor/src/main/java/com/infomaniak/lib/richhtmleditor/JsBridge.kt (hunk 1 above)
Comment (LunarX) [reply rich-html-editor/src/main/java/com/infomaniak/lib/richhtmleditor/JsBridge.kt:46]: Is it truly worth the effort to extract these values into constants and have to manage them for a simple `@IntRange`? For other parts of the code, I agree, but the IntRange I'm not so sure

Comment (KevinBoulongne) [reply rich-html-editor/src/main/java/com/infomaniak/lib/richhtmleditor/JsBridge.kt:49]: At least a comment to explain what are 1 & 7 ?

1 what ? 1 px ? 1 dp ? 1 size ?
7 what ?
Why only 1 to 7 ?
What is the meaning of life ?

File: rich-html-editor/src/main/java/com/infomaniak/lib/richhtmleditor/RichHtmlEditorWebView.kt [hunk 4]
```diff
@@ -117,8 +118,9 @@ class RichHtmlEditorWebView @JvmOverloads constructor(
     fun toggleStrikeThrough() = jsBridge.toggleStrikeThrough()
     fun toggleUnderline() = jsBridge.toggleUnderline()
     fun removeFormat() = jsBridge.removeFormat()
-    fun setTextColor(@ColorInt color: Int) = jsBridge.setTextColor(color)
-    fun setTextBackgroundColor(@ColorInt color: Int) = jsBridge.setTextBackgroundColor(color)
+    fun setTextColor(@ColorInt color: Int) = jsBridge.setTextColor(JsColor(color))
+    fun setTextBackgroundColor(@ColorInt color: Int) = jsBridge.setTextBackgroundColor(JsColor(color))
+    fun setFontSize(@IntRange(from = 1, to = 7) fontSize: Int) = jsBridge.setFontSize(fontSize)
```
Comment (KevinBoulongne) [inline rich-html-editor/src/main/java/com/infomaniak/lib/richhtmleditor/RichHtmlEditorWebView.kt:123]: Same magic numbers, and now they are duplicated.

**Review by KevinBoulongne — CHANGES_REQUESTED** [review]

File: rich-html-editor/src/main/java/com/infomaniak/lib/richhtmleditor/RichHtmlEditorWebView.kt (hunk 4 above)
Comment (LunarX) [reply rich-html-editor/src/main/java/com/infomaniak/lib/richhtmleditor/RichHtmlEditorWebView.kt]: In the end you were not the only one who wanted to extract it anyway, so I did

**Review by KevinBoulongne — APPROVED** [review]

**Pull request merged**